### DISCUSSION TITLE
Adding custom townblock types in config.yml breaks the plugin?

### DISCUSSION THREAD

**Discussion by UNICORNSPOON** [TownyAdvanced/Towny#6490] [Q&A] [answered]
Hi guys,

So it says in the config file I should be able to add custom townblock types... but when I do it breaks the plugin. All I'm doing in copy-pasting one of the existing townblock types and changing the name and map key, but it doesn't work... am 
I missing something? doing something wrong?

**Comment by LlmDl** [comment, marked as answer]
I think you're missing something, since that works fine if you do it right. File an issue ticket instead of making a discussion.

**Reply by UNICORNSPOON** [reply]
Well, can you explain how to do it then?

**Reply by LlmDl** [reply]
https://github.com/TownyAdvanced/Towny/issues/new?assignees=&labels=bug&template=bug_report.yml
